### PR TITLE
Refactor FAB with onScroll animation

### PR DESCRIPTION
**Pull request by jamilxt**
The large FAB looks weird with the design. So I've refactored it to use Extended FAB and added animation onScroll like Gmail does. 

### Old:

We can also consider showing/hiding the Search Bar & Bottom Navigation Bar on the scroll. I will try to do it next if this PR gets merged.
Thank you.

### CODE REVIEW

File: Reply/app/src/main/java/com/example/reply/ui/ReplyListContent.kt [hunk 1]
```diff
@@ -113,20 +119,17 @@ fun ReplyInboxScreen(
...
-                    contentColor = MaterialTheme.colorScheme.onTertiaryContainer
-                ) {
-                    Icon(
-                        imageVector = Icons.Default.Edit,
-                        contentDescription = stringResource(id = R.string.compose),
-                        modifier = Modifier.size(28.dp)
-                    )
-                }
+                    contentColor = MaterialTheme.colorScheme.onTertiaryContainer,
+                    expanded = emailLazyListState.isScrollingUp()
```
Comment (bentrengrove) [inline Reply/app/src/main/java/com/example/reply/ui/ReplyListContent.kt:131]: No need for an extension here, you can replace this with
```suggestion
                    expanded = emailLazyListState.lastScrolledBackward ||
                            !emailLazyListState.canScrollBackward
```

**Review by bentrengrove — CHANGES_REQUESTED** [review]
Thanks for the submission! I am just checking with our designer whether this meets the Material spec that this sample is demonstrating

One minor comment as well, there is an easier way to determine if the fab should be expanded

File: Reply/app/src/main/java/com/example/reply/ui/ReplyListContent.kt (hunk 1 above)
Comment (jamilxt) [reply Reply/app/src/main/java/com/example/reply/ui/ReplyListContent.kt]: Thank you! I really appreciate it. It's been exciting to try Android development again after 4 years!

**Review by bentrengrove — APPROVED** [review]
LGTM!

**Comment by bentrengrove** [comment]
Looks like it just needs formatting. You should be able to just run

`./gradlew --init-script buildscripts/init.gradle.kts spotlessApply`

**Comment by jamilxt** [comment]
> Looks like it just needs formatting. You should be able to just run
> 
> `./gradlew --init-script buildscripts/init.gradle.kts spotlessApply`

I've applied spotless to format the code. 
Thank you.

**Pull request merged**